templates/expo-template-bare-minimum/android/app/build.gradle

```
@@ -13,6 +21,7 @@ react {
     entryFile = file(["node", "-e", "require('expo/scripts/resolveAppEntry')", projectRoot, "android", "absolute"].execute(null, rootDir).text.trim())
     reactNativeDir = new File(["node", "--print", "require.resolve('react-native/package.json')"].execute(null, rootDir).text.trim()).getParentFile().getAbsoluteFile()
     hermesCommand = new File(["node", "--print", "require.resolve('react-native/package.json')"].execute(null, rootDir).text.trim()).getParentFile().getAbsolutePath() + "/sdks/hermesc/%OS-BIN%/hermesc"
+    debuggableVariants = expoDebuggableVariants
```

```suggestion
    debuggableVariants = System.getenv('EX_UPDATES_NATIVE_DEBUG') == "1" ? [] : ['debug']
```
maybe minimize the change to template.

---

I'm going to investigate further to see if this can be done without a template change